addons/ofxOpenCv/scripts/formulas/opencv.sh

```diff
@@ -494,10 +582,10 @@ function build() {
 function copy() {
 
   # prepare headers directory if needed
-  mkdir -p $1/include
+  mkdir -p $1/../../addons/ofxOpenCv/libs/opencv/include
```

Comment:
why this? it was working in other platforms as it was. perhaps we need to fix the path passed to this function in windows?